### ISSUE TITLE
Could you add the feature to enable night light at predefined hours?

### ISSUE THREAD

**Issue by gtumanyan** [pratyakshm/WinRice#66]
### WinRice principles

- [X] I have read the WinRice principles and I believe that this feature request abides by the WinRice principles to my truest sense.

### Explain the feature.

Night light shows warmer colors which helps to fall asleep.

### Does your feature request involve usage of third party software?

No

**Comment by pratyakshm** [comment]
Windows allows you to schedule night light, not sure why you'd want that in WinRice:
![image](https://user-images.githubusercontent.com/54220235/213536644-8cfb1924-2209-4850-aee6-e0efca6f8e71.png)

But I'll think about the request.

**Comment by gtumanyan** [comment]
It's off by default. It would be pretty nice to have it scheduled for the night hours.

**Comment by pratyakshm** [comment]
It would be nicr, sure, but would also result in users spending more time configuring the program before it runs.

**Comment by gtumanyan** [comment]
Maybe make this option Off by default?

**Comment by pratyakshm** [comment]
You mean you want to make the function call commented?

**Comment by gtumanyan** [comment]
Yes so experienced users could uncomment and set the desired scheduled hours.
